.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💡 Feature request
+    url:     url: https://github.com/zkSNACKs/WalletWasabi/discussions/categories/ideas-contributions
+    about: Missing something in Wasabi? Let us know.
+  - name: 📃 User documentation
+    url: https://docs.wasabiwallet.io/
+    about: Search for any questions you have about Wasabi.
```

The documentation is not only question, how about:

```suggestion
    about: Search for any information about Wasabi.
```